lib/ExpensiMark.ts

```diff
@@ -303,7 +303,7 @@ export default class ExpensiMark {
             {
                 name: 'reportMentions',
 
-                regex: /(?<![^ \n*~_])(#[\p{Ll}0-9-]{1,99})(?![^<]*(?:<\/pre>|<\/code>|<\/a>))/gimu,
+                regex: /(?<![^ \n*~_-])(#[\p{Ll}0-9]{1,99})(?![^<]*(?:<\/pre>|<\/code>|<\/a>))/gimu,
```

Comment:
I remove the minus sign - after `0-9` because it will match any word containing minus sign - meaning

If we not remove the minus sign `-` and we type `#user1-#user-2` the output will be:
`#user1-    #user2`

After removing:
`#user1    #user2`

Reply:
But i think if we remove the `-` minus sign will cause other issue like:
typing `#room-test` the mention-report tag will only wrap `#room`

But if we not remove the `-` minus sign it will show like this `#user1-    #user2` if we type `#user1-#user2`

Reply:
Tests are failling actually

Reply:
> Yes because the minus sign is removed after `0-9`

@NJ-2020 It does not look correct to remove the hyphen.